KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+- The act of voting and the gain of voting power is different. 
+
+Governance Council members can exercise the right to vote based on the staking amount. Currently, each GC member receives at least 1 voting right regardless of the number of KLAY they hold. Recognizing a tendency that the member with more KLAY makes a decision that benefits the Klaytn ecosystem in the long run, we are granting a higher voting power to members with more KLAY. Yet, to prevent a particular subject from making an arbitrary decision, we are placing a cap on the voting power one can receive. 
+Therefore, the GC member will receive 1 vote per a certain amount of staked KLAY (initial configuration: 500 million KLAY). The maximum number of votes a GC member can own is one less than the total number of members. In other words, [Maximum Voting Power =  Total number of GC members - 1]. For example, if there are 35 GC members, one can own a maximum of 34 voting power. The 1 KLAY: 1 vote structure prevents the tragedy of the commons and potential monopolies by limiting the maximum number of votes cast. 
+The foundation will provide `votingDelay` (initial configuration: 7 days)  of the notice period and ‘votingPeriod’ of the voting period (initial configuration: 7 days). The list of eligible voters is finalized at the start of the notice period. The qualified electors and voting power of those will be counted at the start of the voting period. The foundation or GC can freely propose and discuss on the forum. Once the discussion is held on the forum, the foundation will put the topic to a vote on the governance portal. 
+The newly implemented governance portal includes the following functions: 
+- Ability to propose and vote on a variety of opinions
+- Information about Governance Council members who are working together for the Klaytn ecosystem
+
+The Klaytn governance voting system will be conducted on smart contracts. The contract design is influenced by Compound Finance and OpenZeppelin. The accounts interacting with the contracts can have proposer, voter, or executor roles.
+A proposer submits and initiates governance proposals. The voting timeline starts with the submission of proposals to the voting contract. A proposal describes the changes to be made on the Klaytn blockchain.
+A proposal not only contains textual descriptions of the change, but it may also include  specifications of attached transactions, or actions. The actions are executed on behalf of a voting contract after the proposal passes. The actions extend the impact of proposals from simply gathering opinions to automatically implementing changes on the blockchain.
+A voter casts votes on the proposal during the voting period. A voter can vote for yes, no, or abstain with all the votes assigned to the voter. The list of voters for each proposal is determined at the proposal submission. The number of votes is determined by the amount of staked KLAYs at the start of the voting period.
```

Comment:
Unclear when the voting period starts.
If we have two proposals, what is the voting period?
- The first proposal submitted on Sep. 20th. -> voting period: 20th ~ 27th
- The second proposal submitted on Sep. 21st -> voting period: 21th ~ 28th
Is this right?

Reply:
@kjhman21 There will be 7days of notice period, and then 7days of voting period.
- First proposal submitted on Sep 10th 00:00
  - notice period 10th 00:00 to 17th 00:00, voting period 17th 00:00 to 24th 00:00
- Second proposal submitted on Sep 11th 00:00
  - notice period 11th 00:00 to 18th 00:00, voting period 18th 00:00 to 25th 00:00.
Please refer to "Voting contract" section.